sound/soc/sof/sof-priv.h

```diff
@@ -323,8 +330,9 @@ struct snd_sof_dev {
 
 	/* power states related */
 	enum sof_d0_substate d0_substate;
-	/* flag to track if the intended power target of suspend is S0ix */
-	bool s0_suspend;
+
+	/* Intended power target of system suspend */
+	enum sof_system_suspend_state system_suspend_target;
```

Comment:
Hmm, this feels a bit out of place as a very specific state of the PM flow internals, in this generic SOF struct. But I guess ok -- at least naming  is aligned with the ACPI API.